blocks/hero-block/features/hero/default.jsx

```diff
@@ -34,20 +42,95 @@ function Hero({ customFields }) {
 		subHeadline,
 		variant = "dark",
 	} = customFields;
-
+	const { arcSite } = useFusionContext();
+	const { fallbackImage } = getProperties(arcSite);
+	const imageId = imageDesktopURL
+		? imageDesktopURL.split("/").pop().split(".").shift()
+		: desktopImgId;
+	const imageMobileId = imageMobileURL
+		? imageMobileURL.split("/").pop().split(".").shift()
+		: imgMobileId;
+	let desktopAuth = useContent(
+		imageId
+			? {
+					source: "signing-service",
+					query: { id: imageId },
+			  }
+			: {}
+	);
+	const imageAuthTokenObj = {};
+	if (desktopAuth?.hash) {
+		desktopAuth[RESIZER_TOKEN_VERSION] = desktopAuth.hash;
+	}
+	if (desktopAuth && !Object.keys(desktopAuth).length) {
+		desktopAuth = imageDesktopAuth;
+	}
+	let mobileAuth = useContent(
+		imageMobileId
+			? {
+					source: "signing-service",
+					query: { id: imageMobileId },
+			  }
+			: {}
+	);
+	const imageMobileAuthTokenObj = {};
+	if (mobileAuth?.hash) {
+		mobileAuth[RESIZER_TOKEN_VERSION] = mobileAuth.hash;
+	}
+	if (mobileAuth && !Object.keys(mobileAuth).length) {
+		mobileAuth = imageMobileAuth;
+	}
 	const classes = [
 		BLOCK_CLASS_NAME,
 		`${BLOCK_CLASS_NAME}--${layout}`,
 		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
 	].join(" ");
 
+	const alt = headline || description || null;
+	const desktopImageParams =
+		imageId && imageDesktopURL
+			? {
+					src: imageANSToImageSrc({
+						_id: imageId,
+						url: imageDesktopURL,
+					}),
+					resizedOptions: {
+						auth: desktopAuth || imageAuthTokenObj,
```

Comment:
The `Source` component is expecting `auth` to be a string, both `desktopAuth` and `imageAuthTokenObj` are objects. We just need `.hash` from them.

Reply:
Fixed auth